src/jacobians.jl

```diff
@@ -232,9 +230,7 @@ function finite_difference_jacobian(
 
         if jac_prototype isa Nothing && sparsity isa Nothing
             J = mapreduce(calculate_Ji_central, hcat, 1:maximum(colorvec))
-            if maximum(colorvec) == 1
-                J = reshape(J, 1, 1)
-            end
+            J = reshape(J, (axes(J, 1), axes(J, 1)))
```

Comment:
looks like this assumes square?

Reply:
Yes. Bummer. StaticArrays are hard. I'll try to come up with a solution.

Reply:
Should be fixed now